### PR TITLE
Align action list

### LINKED ISSUE #304: Frontend: Align Actions list and 'Add action button' on each side site

### PR DESCRIPTION
**Pull request by ReedOnly**
closes #304

### CODE REVIEW

File: frontend/src/components/AnswerSummaryButton.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```
Comment (nicholasdalhaug) [inline frontend/src/components/AnswerSummaryButton.tsx:1]: We need to get Codacy to stop complaining about this.

File: frontend/src/views/Evaluation/FollowUp/FollowUpView.tsx [hunk 2]
```diff
@@ -88,15 +89,21 @@ const FollowUpView = ({ evaluation }: FollowUpViewProps) => {
...
-                                    questionNumber={idx + 1}
-                                    question={question}
-                                    answer={answer}
-                                    disabled={disableAllUserInput || isParticipantCompleted}
-                                    onQuestionSummarySelected={onQuestionSummarySelected}
-                                    viewProgression={viewProgression}
-                                />
-                                <QuestionActionsListWithApi question={question} />
+                                <Box display="flex">
+                                    <Box>
```
Comment (nicholasdalhaug) [inline frontend/src/views/Evaluation/FollowUp/FollowUpView.tsx:93]: Maybe flexgrow here or is this expected behaviour?
image

```suggestion
                                    <Box flexGrow={1}>
```

**Review by nicholasdalhaug — COMMENTED** [review]
Should the commits be squashed? I see that the things the first commit do are changed in the next commit.

File: frontend/src/views/Evaluation/FollowUp/FollowUpView.tsx (hunk 2 above)
Comment (ReedOnly) [reply frontend/src/views/Evaluation/FollowUp/FollowUpView.tsx:93]: agree

File: frontend/src/components/AnswerSummaryButton.tsx (hunk 1 above)
Comment (ReedOnly) [reply frontend/src/components/AnswerSummaryButton.tsx:1]: fixed!

**Review by nicholasdalhaug — APPROVED** [review]

**Pull request merged**